rmf_site_format/src/workcell.rs

```diff
@@ -89,7 +90,14 @@ pub struct Mass(f32);
 
 #[derive(Serialize, Deserialize, Debug, Default, Clone)]
 #[cfg_attr(feature = "bevy", derive(Component))]
-pub struct Inertia {}
+pub struct Inertia {
```

Comment:
Having structs named `Inertial` and `Inertia` doesn't make a lot of sense imo.

I'd recommend changing the current `Inertial` to `Inertia` and change this `Inertia` struct to [`Moment`](https://en.wikipedia.org/wiki/Moment_of_inertia).

In general, the word "inertia" is meant to refer to a body's resistance to acceleration. So it's reasonable to bundle up mass, center of mass, and moment of inertia into one `Inertia` struct. "**Moment** of inertia" refers to resistance to rotational acceleration, so I think it makes sense to use "moment" as its keyword.

Reply:
https://github.com/open-rmf/rmf_site/pull/177/commits/9405bdc2192af3290d99cdc2ccff0cbdff5f2ea7 for Inertia -> Moment and https://github.com/open-rmf/rmf_site/pull/177/commits/422a610a09b6e392d9242f09110e2ed480753a89 for Inertial to Inertia